crates/bitwarden-exporters/src/models.rs

```diff
@@ -14,13 +16,18 @@ impl TryFrom<FolderView> for crate::Folder {
     }
 }
 
-impl TryFrom<CipherView> for crate::Cipher {
-    type Error = MissingFieldError;
+impl crate::Cipher {
+    pub(crate) fn from_cipher(
```

Comment:
This is really unfriendly. And we should figure out a better strategy for handling this.

@dani-garcia Long term we've discussed wanting dedicated DTOs for ciphers. Should we go ahead with it and create a CreateCipherView dto that lets you have a full passkey unencrypted?

FYI @coroiu

Reply:
Changed to use the `set_new_fido2_credentials`. I think we're good for now.